### DISCUSSION TITLE
用Python做自動化程式交易

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#545] [General]
影片：https://youtu.be/xfzGZB4HhEE
程式碼：https://github.com/nickmccullum/algorithmic-trading-python

我也還在學習，忍不住先分享給大家，之後再跟大家分享學習心得喔~

**Comment by Sazmafuji** [comment]
感謝老師分享，這看起來很有趣

**Comment by liangyinglly** [comment]
之前好像有聽說在自動化程式交易中會使用到Selenium來下單？（不是很確定我有沒有記錯）

**Comment by hapiwangy** [comment]
原來老師對這方面的東西也有研究，期待自己也能趕快吸收之後活用

**Comment by SamTU0222** [comment]
原來老師在研究這個啊 希望我在學習python後也可以慢慢活用它

**Comment by yuhao112** [comment]
也有看到很多人說過類似的，如今終於看到真正案例，真是太感謝老師了

**Comment by Lai27330** [comment]
謝謝老師的分享 感覺可以跟科系所學結合 希望未來自己也能夠利用python對自動化交易有更深入的研究

**Comment by ghost** [comment]
好複雜的感覺，老師加油w
